Application/EdFi.Ods.Api/Startup/OdsStartupBase.cs

```diff
@@ -196,6 +198,37 @@ public void ConfigureServices(IServiceCollection services)
             }
 
             services.AddHealthCheck(Configuration.GetConnectionString("EdFi_Admin"), IsSqlServer(databaseEngine));
+
+            ConfigureScheduledJobs(services);
+        }
+
+        // The purpose of the ConfigureScheduleJobs method is to wire-up background tasks to be handled by quartz-scheduler.net
+        private void ConfigureScheduledJobs(IServiceCollection services)
+        {
+            var enabledDistinctScheduledJobs =
+                ApiSettings.ScheduledJobs
+                    .Where(a => a.IsEnabled)
+                    .GroupBy(a => a.Name)
+                    .Select(a => a.First()).ToList();
+
+            if (!enabledDistinctScheduledJobs.Any())
+            {
+                return;
+            }
+
+            IScheduledJobsConfiguratorFactory scheduledJobsConfiguratorFactory = new ScheduledJobsConfiguratorFactory();
```

Comment:
@bptillman Hey Ben, just for context here is where I am trying to replace the ScheduledJobsConfiguratorFactory with autofac if I can.  However, at this point in the code execution flow, the autofac container has not yet been loaded